vital/algo/estimate_fundamental_matrix.h

```diff
@@ -12,66 +12,84 @@
 
 #include <vital/vital_config.h>
 
-#include <vector>
 #include <memory>
+#include <vector>
 
 #include <vital/algo/algorithm.h>
 #include <vital/types/feature_set.h>
-#include <vital/types/match_set.h>
 #include <vital/types/fundamental_matrix.h>
+#include <vital/types/match_set.h>
 
 namespace kwiver {
+
 namespace vital {
+
 namespace algo {
 
-/// An abstract base class for estimating a fundamental matrix from matching 2D points
+/// An abstract base class for estimating a fundamental matrix from matching 2D
+/// points
 class VITAL_ALGO_EXPORT estimate_fundamental_matrix
-  : public kwiver::vital::algorithm_def<estimate_fundamental_matrix>
+  : public kwiver::vital::algorithm_def< estimate_fundamental_matrix >
 {
 public:
+
```

Comment:
This looks like a bug. What Uncrustify are you using?

Reply:
Fixed.